zhmccli/_cmd_cpc.py

```diff
@@ -323,6 +324,12 @@ def get_em_data(cmd_ctx, cpc):
               required=False, default=DEFAULT_DPM_FORMAT,
               help='Controls in which data format the DPM configuration file '
               'is written. Default: {d}.'.format(d=DEFAULT_DPM_FORMAT))
+@click.option('--exclude-extra-fields', '-e', is_flag=True, default=False,
```

Comment:
Why not call it `--exclude-meta-information`? Would be good to clean up nomenclature with the description. "extra information", "additional information", "additional meta information" are all used.

Reply:
Excellent idea!